apps/extension/background-script.ts

```diff
@@ -20,10 +21,9 @@ import {
 import { createDAppNotificationResponse, getHexChanId } from './src/utils/network.utils';
 import { getState } from './src/utils/state.utils';
 
-const APP_LOCK_PERIOD = 5 * 60 * 1000;
-
-let passwordHash = INITIAL_PASSWORD_HASH;
-let lastUserActivityTimestamp = 0;
+const LOGIN_TOKEN = 'login_token';
+const LOCK_TIME_PERIOD = 'lock_time_period';
+const LAST_USER_ACTIVITY_TIMESTAMP = 'last_user_activity_timestamp';
```

Comment:
add `_KEY` to the end of these variables.  (as this's the keys where data is stored)  
and rename `LOGIN_TOKEN` to `PASSWORD_HASH_KEY` - name should represent what data is stored there